clio-client/src/test/scala/org/broadinstitute/clio/client/dispatch/MoveGvcfSpec.scala

```diff
@@ -97,18 +84,45 @@ class MoveGvcfSpec extends BaseClientSpec {
   it should "throw an exception if the destination path is not in GCP" in {
     recoverToSucceededIf[Exception] {
       val command = MoveGvcf(
-        transferGvcfV1Key = testGvcfTransferV1Key,
+        key = testGvcfTransferV1Key,
         destination = "/this/is/a/local/path"
       )
       succeedingDispatcher.dispatch(command)
     }
   }
 
+  it should "throw an exception if given a non-directory destination for multiple source files" in {
+    recoverToSucceededIf[Exception] {
```

Comment:
You could make this a more specific check, to avoid false positives if an exception is thrown for some other reason.